com.chartboost.helium/Documentation/integration/integration-checker.md

```diff
@@ -0,0 +1,36 @@
+# Helium Integration Checker
+
+`HeliumIntegrationChecker.cs` is a utility class introduced in Helium Unity 3.3.0. The `HeliumIntegrationChecker` provides common utility methods to make developers life easier when integrating, updating, and testing the Helium Unity SDK. The following functionality can be found in it, and it is accessible through Unity Context Menus:
+
+* *`Helium/Integration/Status Check`*
+
+This will run a series of checks and test inside the Helium Unity SDK. It attempts to corroborate that any common integration issues are surfaced to developers with a popup window. It is recommended to developers to run this method once after every Helium Unity SDK update, as it will raise any issues that might need correction after package update is completed.
+
+* *`Helium/Integration/UnityAds Check`*
+
+This will run a UnityAds integration specific check, it is completely optional and its only purpose is to make sure your integration is aligned with Helium / UnityAds integration standards. Whether you integrate the UnityAds SDK through the Unity Package Manager, or throug the Helium Optional dependency, this check will make sure that any warning are mentioned to developers through a popup window.
+
+* *`Helium/Integration/Force Reimport Adapters`*
+
+This will automatically reimport all of your existing Ad Adapters. This utility function will double check this is an intended action. Reimporting Ad Adapters is particularly useful in scenarios where you update your Helium Unity SDK and want to make sure all of your ad adapters are up to date. On the other hand, if you were to downgrade your Helium Unity SDK, this could help you get that version specific dependencies in a much more efficient manner.
```

Comment:
```suggestion
This will automatically reimport all of your existing Ad Adapters. This utility function will double-check this is an intended action. Reimporting Ad Adapters is particularly useful in scenarios where you update your Helium Unity SDK and want to make sure all of your ad adapters are up to date. On the other hand, if you were to downgrade your Helium Unity SDK, this could help you get that version specific dependencies in a much more efficient manner.
```